examples/one-page.html

```diff
@@ -0,0 +1,257 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <meta http-equiv="X-UA-Compatible" content="ie=edge" />
+    <title>Spectacle One-Page Example</title>
+  </head>
+  <body>
+    <div id="root"></div>
+
+    <script src="https://unpkg.com/react@16.12.0/umd/react.production.min.js"></script>
+    <script src="https://unpkg.com/react-dom@16.12.0/umd/react-dom.production.min.js"></script>
+    <script src="https://unpkg.com/react-is@16.12.0/umd/react-is.production.min.js"></script>
+    <script src="https://unpkg.com/prop-types@15.7.2/prop-types.min.js"></script>
+    <script src="https://unpkg.com/spectacle@^6.0.0-alpha/dist/spectacle.min.js"></script>
```

Comment:
OPTIONAL: Do we want to preemptively change this to:

```
<script src="https://unpkg.com/spectacle@^6/dist/spectacle.min.js"></script>
```

? It won't work until we publish, but I think that will happen quickly after anyways.